src/sentry/api/endpoints/project_group_index.py

```diff
@@ -680,6 +683,13 @@ def put(self, request, project):
                         group=group,
                         sender=type(self),
                     )
+                else:
+                    issue_resolved.send_robust(
```

Comment:
I think @macqueen is working on the organization level bulk update APIs. We should take care to ensure this code ends up there as well.